server.js

```diff
@@ -0,0 +1,39 @@
+npm i express multer cors nodemon –save
+
+var express = require('express');
```

Comment:
What's the difference between this express app and the other one?

Reply:
ah so this one was something I was using for the upload form, but if we're gonna leave it to backend then it's not really necessary to keep